pillarbox-analytics/docs/README.md

```diff
@@ -40,6 +40,35 @@ class DemoApplication : Application() {
 }
 ```
 
+### Handle user consent
+User consent can be configured at initialization:
+
+```kotlin
+val initialUserConsent = UserConsent(
+    comScore = ComScoreUserConsent.UNKNOWN,
+    commandersActConsentServices = emptyList()
+    )
+
+val config = AnalyticsConfig(
+    vendor = AnalyticsConfig.Vendor.SRG,
+    nonLocalizedApplicationName = "PillarboxDemo",
+    appSiteName = "pillarbox-demo-android",
+    sourceKey = AnalyticsConfig.SOURCE_KEY_SRG_DEBUG,
+    userConsent = initialUserConsent
+)
+
+initSRGAnalytics(config = config)
```

Comment:
No because `initSRGAnalytics ` is an extension of `Application`. An other way to initialize, is to call `SRGAnalytics.init(application = this, config = config)`.